packages/integration/src/processVanillaFile.ts

```diff
@@ -82,21 +83,21 @@ export function processVanillaFile({
   const cssImports = [];
 
   for (const [serialisedFileScope, fileScopeCss] of cssByFileScope) {
-    const filescope = parseFileScope(serialisedFileScope);
```

Comment:
You'll need to add a changeset (`yarn changeset`) to make sure the `integration` package is released with these changes.